plugins/cilium-cni/cilium-cni.go

```diff
@@ -37,7 +39,6 @@ import (
 	_ "github.com/cilium/cilium/plugins/cilium-cni/chaining/generic-veth"
 	_ "github.com/cilium/cilium/plugins/cilium-cni/chaining/portmap"
 	"github.com/cilium/cilium/plugins/cilium-cni/types"
-	"github.com/cilium/ebpf"
 
 	"github.com/containernetworking/cni/pkg/skel"
```

Comment:
The ordering here seems to have diverged from the rest of the PR / the description.

Reply:
Thanks, fixed. I'm not sure why neither `goimports` nor `golangci-lint` missed this.